### ISSUE TITLE
loss goes up 

### ISSUE THREAD

**Issue by 123liluky** [JiahuiYu/generative_inpainting#133]
I trained my 1883 images about 380 epochs with parameters like below:
#parameters
DATASET: 'celebahq'
RANDOM_CROP: False
VAL: False
LOG_DIR: 'logs'
MODEL_RESTORE: 'a'  #

GAN: 'wgan_gp'  # 'dcgan', 'lsgan', 'wgan_gp', 'one_wgan_gp'
PRETRAIN_COARSE_NETWORK: False # unused
GAN_LOSS_ALPHA: 0.001  # dcgan: 0.0008, wgan: 0.0005, onegan: 0.001
WGAN_GP_LAMBDA: 10
COARSE_L1_ALPHA: 0.5
L1_LOSS_ALPHA: 0.5
AE_LOSS_ALPHA: 0.5
GAN_WITH_MASK: False
DISCOUNTED_MASK: True
RANDOM_SEED: False
PADDING: 'SAME'

#training
NUM_GPUS: 1
GPU_ID: 3  # -1 indicate select any available one, otherwise select gpu ID, e.g. [0,1,3]
TRAIN_SPE: 940
MAX_ITERS: 400000
VIZ_MAX_OUT: 10
GRADS_SUMMARY: False
GRADIENT_CLIP: False
GRADIENT_CLIP_VALUE: 0.1
VAL_PSTEPS: 50

#data
DATA_FLIST:
  celebahq: [
    'training_data/train_shuffled.flist',
    ''  # VAL=False 
  ]

STATIC_VIEW_SIZE: 30
IMG_SHAPES: [224, 224, 3]
HEIGHT: 224     
WIDTH: 224   
MAX_DELTA_HEIGHT: 0
MAX_DELTA_WIDTH: 0
BATCH_SIZE: 2
VERTICAL_MARGIN: 0
HORIZONTAL_MARGIN: 0

#loss
AE_LOSS: True
L1_LOSS: True
GLOBAL_DCGAN_LOSS_ALPHA: 1.
GLOBAL_WGAN_LOSS_ALPHA: 1.

#loss legacy
LOAD_VGG_MODEL: False
VGG_MODEL_FILE: data/model_zoo/vgg16.npz
FEATURE_LOSS: False
GRAMS_LOSS: False
TV_LOSS: False
TV_LOSS_ALPHA: 0.
FEATURE_LOSS_ALPHA: 0.01
GRAMS_LOSS_ALPHA: 50 # unused
SPATIAL_DISCOUNTING_GAMMA: 0.9
I found loss kept going up.
![image](https://user-images.githubusercontent.com/20832778/45611513-37dcaa00-ba92-11e8-9acb-7e166892a7ce.png)
And imges with bigger masks were inpainted unsatisfing.
![image](https://user-images.githubusercontent.com/20832778/45611807-50998f80-ba93-11e8-8802-ec3861b0bfac.png)
![image](https://user-images.githubusercontent.com/20832778/45611865-99e9df00-ba93-11e8-854d-35566e56ff46.png)
Could you please give me some help?

**Comment by JiahuiYu** [comment]
A loss look like this is fine. The performance is not good because of many other factors. One potential problem is that the dataset you have is too small. Please have a reference of issue #75.

**Comment by an1018** [comment]
> I trained my 1883 images about 380 epochs with parameters like below:
> #parameters
> DATASET: 'celebahq'
> RANDOM_CROP: False
> VAL: False
> LOG_DIR: 'logs'
> MODEL_RESTORE: 'a' #
> 
> GAN: 'wgan_gp' # 'dcgan', 'lsgan', 'wgan_gp', 'one_wgan_gp'
> PRETRAIN_COARSE_NETWORK: False # unused
> GAN_LOSS_ALPHA: 0.001 # dcgan: 0.0008, wgan: 0.0005, onegan: 0.001
> WGAN_GP_LAMBDA: 10
> COARSE_L1_ALPHA: 0.5
> L1_LOSS_ALPHA: 0.5
> AE_LOSS_ALPHA: 0.5
> GAN_WITH_MASK: False
> DISCOUNTED_MASK: True
> RANDOM_SEED: False
> PADDING: 'SAME'
> 
> #training
> NUM_GPUS: 1
> GPU_ID: 3 # -1 indicate select any available one, otherwise select gpu ID, e.g. [0,1,3]
> TRAIN_SPE: 940
> MAX_ITERS: 400000
> VIZ_MAX_OUT: 10
> GRADS_SUMMARY: False
> GRADIENT_CLIP: False
> GRADIENT_CLIP_VALUE: 0.1
> VAL_PSTEPS: 50
> 
> #data
> DATA_FLIST:
> celebahq: [
> 'training_data/train_shuffled.flist',
> '' # VAL=False
> ]
> 
> STATIC_VIEW_SIZE: 30
> IMG_SHAPES: [224, 224, 3]
> HEIGHT: 224
> WIDTH: 224
> MAX_DELTA_HEIGHT: 0
> MAX_DELTA_WIDTH: 0
> BATCH_SIZE: 2
> VERTICAL_MARGIN: 0
> HORIZONTAL_MARGIN: 0
> 
> #loss
> AE_LOSS: True
> L1_LOSS: True
> GLOBAL_DCGAN_LOSS_ALPHA: 1.
> GLOBAL_WGAN_LOSS_ALPHA: 1.
> 
> #loss legacy
> LOAD_VGG_MODEL: False
> VGG_MODEL_FILE: data/model_zoo/vgg16.npz
> FEATURE_LOSS: False
> GRAMS_LOSS: False
> TV_LOSS: False
> TV_LOSS_ALPHA: 0.
> FEATURE_LOSS_ALPHA: 0.01
> GRAMS_LOSS_ALPHA: 50 # unused
> SPATIAL_DISCOUNTING_GAMMA: 0.9
> I found loss kept going up.
> ![image](https://user-images.githubusercontent.com/20832778/45611513-37dcaa00-ba92-11e8-9acb-7e166892a7ce.png)
> And imges with bigger masks were inpainted unsatisfing.
> ![image](https://user-images.githubusercontent.com/20832778/45611807-50998f80-ba93-11e8-8802-ec3861b0bfac.png)
> ![image](https://user-images.githubusercontent.com/20832778/45611865-99e9df00-ba93-11e8-854d-35566e56ff46.png)
> Could you please give me some help?

@123liluky I am the beginner of Tensorflow, can you share the code for drawing curves?Thank you very much！
